iterating-over-data/iterating-and-special-cases.md

```diff
@@ -0,0 +1,132 @@
+# Iterating and Special Cases
+
+## Learning Goals
+
+- Define the "break" keyword as a way to exit a loop
+- Define the "continue" keyword as a way to advance an iteration in a loop
+- Use the "range()" syntax combined with a for loop
+
+## Introduction
+
+There are more things we can do with loops!
+
+Are we able to stop a loop before it gets to the last element? Are we able to use `for` loops on other data structures?
+
+By learning these different pieces of syntax, we can learn to wield them wisely whenever it becomes relevant.
+
+## Vocabulary and Synonyms
+
+| Vocab      | Definition                                                       | Synonyms | How to Use in a Sentence                                                                     |
+| ---------- | ---------------------------------------------------------------- | -------- | -------------------------------------------------------------------------------------------- |
+| `break`    | A keyword that will exit an entire loop                          | -        | "We can break out of the loop using `break`," "The loop exited early because it saw `break`" |
+| `continue` | A keyword that will immediately advance one interation in a loop | -        | "The loop moved on because it saw `continue`"                                                |
+| `range()`  | A Python function that creates a sequence                        | -        | "We used a `for` loop and `range()` in order to iterate through a series of numbers.         |
+
+## `break` Exits a Loop
+
+`break` is a keyword that we can put in a `for` loop. When our code comes across `break`, it will exit the `for` loop entirely.
```

Comment:
avoid perspective shift in description of break.
shift voicing to the interpreter hitting the break, rather than our code.
provide a bit more explicit description of what the break does.
provide a common synonymous expression.
```suggestion
`break` is a keyword that can be used in a `for` loop. When the Python interpreter encounters a `break`, it will immediately exit the current loop, skipping any instructions from after the `break` until the end of the loop. We also say that it _breaks out_ of the loop.
```